### DISCUSSION TITLE
continues human detection not based on movement option

### DISCUSSION THREAD

**Discussion by kucau0901** [blakeblackshear/frigate#984] [Ideas]
hi, frigate has been very useful and very solid for my external cameras . these few days ,  i tried frigate for my internal camera and for presence detection using  zones . for example "sofa" zone in living room.  problem is when i sleep on the sofa there will be no detection since no movement and living room fan  a  will be turn off as no movement detected . if there is option  per camera for continues image detection  to coral despite no motion let say every x sec or min  then frigate will be usefull   for room presence detection.   any plan for this?

**Comment by blakeblackshear** [comment]
Duplicate of #841
